### PR TITLE
Update 5 min clearinghouse and Detector Status Models

### PR DESCRIPTION
**Pull request by kengodleskidot**
Updates were made to the detector status and 5-minute agg clearinghouse models as part of QC analysis to include stations that were missing. In the detector status model, the join was updated to include the meta data before the final CTE. The 5-minute agg clearinghouse model now uses the active detectors model to capture all active detectors on a given date.

### CODE REVIEW

**Review by JamesSLogan — APPROVED** [review]

**Review by mmmiah — APPROVED** [review]
Leave a comment for clarification! Otherwise looks good to me!

File: transform/models/intermediate/clearinghouse/int_clearinghouse__detector_agg_five_minutes_with_missing_rows.sql [hunk 1]
```diff
@@ -29,10 +29,8 @@ detector_meta as (
 detector_date_range as (
```
[the remark this answers is not in the document]
Comment (mmmiah) [reply transform/models/intermediate/clearinghouse/int_clearinghouse__detector_agg_five_minutes_with_missing_rows.sql:29]: if you change this date, when everything will be flash out for entire time period, how does that work???

[the remark this answers is not in the document]
Comment (kengodleskidot) [reply transform/models/intermediate/clearinghouse/int_clearinghouse__detector_agg_five_minutes_with_missing_rows.sql:28]: In a future PR I am going to modify this to use the global variable `pems_clearinghouse_start_date` and likely limit the timestamp in the spine based on the min(sample_date) in detector_agg to optimize the model. For now I want to get the PR merged so you can QC the imputation models.

[the remark this answers is not in the document]
Comment (mmmiah) [reply transform/models/intermediate/clearinghouse/int_clearinghouse__detector_agg_five_minutes_with_missing_rows.sql:28]: alright! Thanks!

**Pull request merged**